tests/integration/targets/mso_tenant/tasks/main.yml

```diff
@@ -264,7 +303,7 @@
 - name: Verify nm_add_tenant3 (when mso_username != admin)
   assert:
     that:
-    - nm_add_tenant3.current.userAssociations | length == 2
+    - nm_add_tenant3.current.userAssociations | length == 3
```

Comment:
Shouldn't this remain 2 based on the names of the remote users provided above?

Reply:
no because i added an additional user it is 3

Reply:
I'm definitely missing something here because when: mso_username != 'admin' exists for the test, but the same remote user which is 'ansible_github_ci' was already added. 

Actually Two tasks with the same name '- name: Add tenant 3 (normal_mode))' exists which is a little confusing.